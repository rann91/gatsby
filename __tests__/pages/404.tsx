import { render } from '@testing-library/react'
import React from 'react'
import NotFoundPage from '../../src/pages/404'

describe('404', () => {
  it('renders correctly', () => {
    const { container } = render(<NotFoundPage />)

    expect(container).toMatchSnapshot()
  })
})
