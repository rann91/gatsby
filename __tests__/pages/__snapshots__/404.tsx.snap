// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`404 renders correctly 1`] = `
<div>
  <div
    class="css-1ptauik"
    data-testid="section"
  >
    <div
      class="css-jml5s4-NotFoundPage"
    >
      <h1
        class="css-16rjs26"
      >
        Page not found
      </h1>
      <p
        class="css-1pd4dm3"
      >
        The page you have requested cannot be found.
      </p>
      <div
        class="css-vurnku"
      >
        <a
          class="css-1mygz8v-NotFoundPage"
          data-testid="gatsby-link"
          href="/"
          target="_self"
        >
          Back to home
        </a>
      </div>
    </div>
  </div>
</div>
`;
